projects/adrv9009zu11eg/adrv2crr_fmcomms8/system_bd.tcl

```diff
@@ -0,0 +1,86 @@
+
+ad_ip_instance xlconcat spi1_csn_concat
+ad_ip_parameter spi1_csn_concat CONFIG.NUM_PORTS 3
+ad_connect  sys_ps8/emio_spi1_ss_o_n spi1_csn_concat/In0
+ad_connect  sys_ps8/emio_spi1_ss1_o_n spi1_csn_concat/In1
+ad_connect  sys_ps8/emio_spi1_ss2_o_n spi1_csn_concat/In2
+ad_connect  spi1_csn_concat/dout spi1_csn
+ad_connect  sys_ps8/emio_spi1_sclk_o spi1_sclk
+ad_connect  sys_ps8/emio_spi1_m_o spi1_mosi
+ad_connect  sys_ps8/emio_spi1_m_i spi1_miso
+ad_connect  sys_ps8/emio_spi1_ss_i_n VCC
+ad_connect  sys_ps8/emio_spi1_sclk_i GND
+ad_connect  sys_ps8/emio_spi1_s_i GND
+
+ad_ip_instance axi_gpio axi_fmcomms8_gpio
+ad_ip_parameter axi_fmcomms8_gpio CONFIG.C_IS_DUAL 1
+ad_ip_parameter axi_fmcomms8_gpio CONFIG.C_GPIO_WIDTH 32
+ad_ip_parameter axi_fmcomms8_gpio CONFIG.C_GPIO2_WIDTH 32
+ad_ip_parameter axi_fmcomms8_gpio CONFIG.C_INTERRUPT_PRESENT 1
+
+ad_connect  fmcomms8_gpio0_i axi_fmcomms8_gpio/gpio_io_i
+ad_connect  fmcomms8_gpio0_o axi_fmcomms8_gpio/gpio_io_o
+ad_connect  fmcomms8_gpio0_t axi_fmcomms8_gpio/gpio_io_t
+ad_connect  fmcomms8_gpio1_i axi_fmcomms8_gpio/gpio2_io_i
+ad_connect  fmcomms8_gpio1_o axi_fmcomms8_gpio/gpio2_io_o
+ad_connect  fmcomms8_gpio1_t axi_fmcomms8_gpio/gpio2_io_t
+
+ad_xcvrpll  ref_clk_c util_adrv9009_som_xcvr/qpll_ref_clk_8
+ad_xcvrpll  ref_clk_d util_adrv9009_som_xcvr/cpll_ref_clk_8
+ad_xcvrpll  ref_clk_d util_adrv9009_som_xcvr/cpll_ref_clk_9
+ad_xcvrpll  ref_clk_c util_adrv9009_som_xcvr/cpll_ref_clk_10
+ad_xcvrpll  ref_clk_c util_adrv9009_som_xcvr/cpll_ref_clk_11
+ad_xcvrpll  ref_clk_c util_adrv9009_som_xcvr/qpll_ref_clk_12
+ad_xcvrpll  ref_clk_d util_adrv9009_som_xcvr/cpll_ref_clk_12
+ad_xcvrpll  ref_clk_d util_adrv9009_som_xcvr/cpll_ref_clk_13
+ad_xcvrpll  ref_clk_c util_adrv9009_som_xcvr/cpll_ref_clk_14
+ad_xcvrpll  ref_clk_c util_adrv9009_som_xcvr/cpll_ref_clk_15
+
+ad_xcvrpll  axi_adrv9009_som_tx_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_qpll_rst_8
+ad_xcvrpll  axi_adrv9009_som_rx_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_cpll_rst_8
+ad_xcvrpll  axi_adrv9009_som_rx_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_cpll_rst_9
+ad_xcvrpll  axi_adrv9009_som_obs_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_cpll_rst_10
+ad_xcvrpll  axi_adrv9009_som_obs_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_cpll_rst_11
+ad_xcvrpll  axi_adrv9009_som_tx_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_qpll_rst_12
+ad_xcvrpll  axi_adrv9009_som_rx_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_cpll_rst_12
+ad_xcvrpll  axi_adrv9009_som_rx_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_cpll_rst_13
+ad_xcvrpll  axi_adrv9009_som_obs_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_cpll_rst_14
+ad_xcvrpll  axi_adrv9009_som_obs_xcvr/up_pll_rst util_adrv9009_som_xcvr/up_cpll_rst_15
+
+ad_ip_parameter axi_sysid_0 CONFIG.ROM_ADDR_BITS 9
+ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "[pwd]/mem_init_sys.txt"
+ad_ip_parameter rom_sys_0 CONFIG.ROM_ADDR_BITS 9
+set sys_cstring "sys rom custom string placeholder"
```

Comment:
Can we add here a text that is more specific to the project?

Reply:
I don't think we've used the custom string in any of the current projects. It's supposed to be used for variations of the default project. The project is the default one.